backend/src/v2/compiler/argocompiler/testdata/hello_world.yaml

```diff
@@ -50,7 +50,7 @@ spec:
       - '{{inputs.parameters.kubernetes-config}}'
       command:
       - driver
-      image: gcr.io/ml-pipeline/kfp-driver@sha256:60d6d090937d7de393b3740520f4a08fcf85c3419bbdb10bd266cd446d8fe44a
+      image: gcr.io/ml-pipeline-test/kfp-driver@sha256:577221911abd8e6be65d7afad1bdb8d5e4544a11822489f9f5edb72d6e7f87de
```

Comment:
When this PR is ready for merge, can you build `gcr.io/ml-pipeline/kfp-driver` and `gcr.io/ml-pipeline/kfp-launcher` using these Dockerfile instead?
- https://github.com/kubeflow/pipelines/blob/master/backend/Dockerfile.driver
- https://github.com/kubeflow/pipelines/blob/master/backend/Dockerfile.launcher

Reply:
Thank you, I updated the driver/launcher images.